OpenRA.Mods.Cnc/CncLoadScreen.cs

```diff
@@ -41,7 +43,9 @@ public override void Init(Manifest m, Dictionary<string, string> info)
 			r = Game.Renderer;
 			if (r == null) return;
 
-			sheet = new Sheet(Platform.ResolvePath(loadInfo["Image"]));
+			var image = Platform.ResolvePath(info["Image"]);
+			GlobalFileSystem.Mount(Path.GetDirectoryName(image));
```

Comment:
It is absolute vs. relative path inconsistencies. The game loads and labels a relative one, but the code here wants an absolute one. Never mattered as we simply searched through the whole virtual file system for the one file. This adds redundancy, but saves lookup time.